test/interop/C/exportString/badFormalIntent.good

```diff
@@ -1,5 +1,3 @@
-badFormalIntent.chpl:5: error: Formal 's' of type 'string' in exported routine 'badFormalIntentIn' may not have the 'in' intent
 badFormalIntent.chpl:7: error: Formal 's' of type 'string' in exported routine 'badFormalIntentOut' may not have the 'out' intent
 badFormalIntent.chpl:9: error: Formal 's' of type 'string' in exported routine 'badFormalIntentInOut' may not have the 'inout' intent
 badFormalIntent.chpl:11: error: Formal 's' of type 'string' in exported routine 'badFormalIntentRef' may not have the 'ref' intent
-badFormalIntent.chpl:13: error: Formal 's' of type 'string' in exported routine 'badFormalIntentConstIn' may not have the 'const in' intent
```

Comment:
I think this test was intended to track all the cases that didn't work.  If I understand right, that means it should now track const ref and the default intent?

Reply:
Sure, fixed.